docs/asyncio_blocking_operations.md

```diff
@@ -95,6 +95,22 @@ urllib does blocking I/O and should be run in the executor with the standard met
 
 `os.stat` does blocking disk I/O and should be run in the executor with the standard methods above.
 
+# write_bytes
```

Comment:
```suggestion
#### write_bytes
```
And so on for below but perhaps this whole thing should be changed to a list/table or something instead.

Reply:
I was apparently a lot more exhausted that I thought last night